Makefile

```diff
@@ -70,38 +70,69 @@ start_build:
 $(SITEBUILDDIR)/%.svg: $(SOURCEDIR)/%.svg
 	$(SCOUR) $(SCOUR_FLAGS) $< $@
 
-build: clean start_build build_images build_spec sphinx-build-docs
+build: clean start_build build_images build_spec build_docs
 
 build_images: $(SITEBUILDDIR)/$(OBJECTS)
 
 build_spec: build_spec_rfc build_spec_w3c
 
+# PS and RTF support are provided using a combination of wkhtmltopdf, pdf2ps and/or GNU enscript.
+requirements_xml2rfc:
+	sudo apt install xml2rfc enscript
```

Comment:
I think it's not a good idea to mess installing OS packages in runtime.
First point — it breaks down using this makefile on different OS. For example mac os users like me can not simply run this makefile.
And second point: it's better to put OS requirements in gh-action steps. So it can be effectively cached without run every time.

Reply:
I've tried to improve the PR rgd your comments (pls see last commit). having said that, you can ignore those makefile targets (these are _not_ run as part of actual build targets), they are mainly to help my memory / for convenience (when on linux) ..